src/nidcpower/templates/session.py/fancy_initialize.py.mako

```diff
@@ -0,0 +1,91 @@
+<%page args="f, config, method_template"/>\
```

Comment:
All of the code related to the lock workaround should be clearly marked with TODO comments that point to the issue for removing the workarounds.

The issue itself should contain explicit instructions on what needs to be removed, since we will likely forget the specifics :(

Reply:
47a2cc5e9be4035589af6eaa086acfe79b965db0